src/app/fyle/spender-onboarding/spender-onboarding.page.ts

```
@@ -122,9 +122,10 @@ export class SpenderOnboardingPage {
 
   markStepAsComplete(): void {
     if (this.currentStep === OnboardingStep.CONNECT_CARD) {
-      this.spenderOnboardingService.markConnectCardsStepAsComplete().subscribe(() => {
-        this.currentStep = OnboardingStep.OPT_IN;
-      });
+      this.spenderOnboardingService
+        .markConnectCardsStepAsComplete()
+        .pipe(tap(() => (this.currentStep = OnboardingStep.OPT_IN)))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)